### ISSUE TITLE
Create a guideline for selecting the creator and audit provider scripts

### ISSUE THREAD

**Issue by obulat** [WordPress/openverse-catalog#637]
I think we should open a separate issue to discuss the creator question, because it is going to be a little more complex.

Here's an example where the record is a (delightful) photograph: https://collection.sciencemuseumgroup.org.uk/objects/co8210526/timmy-the-cat-guardian-of-mr-browns-prize-budgerigars-gelatin-silver-print . In the json you can see that the `maker` will correctly attribute the photographer: https://collection.sciencemuseumgroup.org.uk/api/objects/co8210526

And then for @obulat's example, here's the API response: https://collection.sciencemuseumgroup.org.uk/api/objects/co56202. This one is several layers deep, because it's a photograph of a replica of an object created by Galileo Galilei, but he is the only source listed under `maker`. However in the `type` field, the maker type is labeled first as `designer` and also `maker (original)`. Maybe we could use this to detect maker types that aren't true sources we want to identify as 'creator'.

The Board of Trustees _is_ in that API response under `source.legal.rights`, so perhaps that could be a field we fall back to if there isn't an appropriate maker type.

Where possible we should probably keep these refactors as small as we can, though. But this is really interesting and definitely worth addressing -- I'm glad you noticed @obulat, good eye!

_Originally posted by @stacimc in https://github.com/WordPress/openverse-catalog/issues/576#issuecomment-1198611794_

**Comment by AetherUnbound** [comment]
I agree, it sound like we should have multiple "tiers" with fallbacks as we look for the creator within the Science Museum metadata! I like the indexed description idea too, that sounds like it's also something we should look at down the line!
